mscore/inspector/inspector_instrchange.ui

```diff
@@ -30,12 +30,15 @@
     <number>0</number>
    </property>
    <item>
-    <widget class="QFrame" name="frame">
-     <property name="frameShape">
-      <enum>QFrame::HLine</enum>
+    <widget class="Line" name="line">
+     <property name="minimumSize">
+      <size>
+       <width>0</width>
+       <height>0</height>
+      </size>
      </property>
-     <property name="frameShadow">
-      <enum>QFrame::Sunken</enum>
+     <property name="orientation">
+      <enum>Qt::Horizontal</enum>
```

Comment:
Don't know why you made this change, that `QFrame` is used for every other inspector I've seen before.

Reply:
I have no memory of changing this on purpose, so I'm guessing this was something that happened automatically or inadvertently.

Reply:
Then revert this change ;-)

Reply:
I see both used inconsistently in different inspectors, but I believe the Line, being more specialized, is the better way to achieve the result (of a line) than a QFrame set the shape of an HLine.